examples/lib/stories/input/keyboard_component_example.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:flame/input.dart';
+import 'package:flame/palette.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+
+import '../../commons/ember.dart';
+
+class KeyboardComponentExample extends FlameGame
+    with HasKeyboardHandlerComponents {
+  static const String description = '''
+    Similar to the default Keyboard example, but shows a different implementation approach,
+    which uses Flame's KeyboardComponent to handle input.
+    Usage: Use A S D W to steer Ember.
+  ''';
+
+  static final Paint white = BasicPalette.white.paint();
+  static const int speed = 200;
+
+  late final Ember ember;
+  final Vector2 velocity = Vector2(0, 0);
+
+  @override
+  Future<void> onLoad() async {
+    ember = Ember(position: size / 2, size: Vector2.all(100));
+    add(ember);
+
+    add(
+      KeyboardComponent(
+        keyUp: {
+          LogicalKeyboardKey.keyA: _stopX,
+          LogicalKeyboardKey.keyD: _stopX,
+          LogicalKeyboardKey.keyW: _stopY,
+          LogicalKeyboardKey.keyS: _stopY,
```

Comment:
on key up you need to check what other keys are pressed, otherwise this will cause non-ideal behavior.

the more correct impl IMO is:
* user holds A -> goes left
* user holds D -> goes right
* users releases D -> as A is pressed, goes left (not stop, which is the current impl)

I think this a perfect place to have a correct example so we can share with people when they do the naive solution

Reply:
example from [ghost v. wonderland](https://github.com/bluefireteam/ghost-vs-wonderland/blob/master/game/lib/game.dart#L63)

Reply:
Sounds good, will look on ghost game

Reply:
done!